app/views/event/kinds/_precondition_fields.html.haml

```diff
@@ -3,17 +3,26 @@
 
   #precondition_summary.inline{style: 'padding-bottom: 5px;',
                                data: { and: t('event.kinds.qualifications.and'),
-                                       or: t('event.kinds.qualifications.or') } }
-    - entry.grouped_qualification_kind_ids('precondition', 'participant').each_with_index do |ids, index|
+                                 or: t('event.kinds.qualifications.or'),
+                                 validity_valid: t('activerecord.attributes.event/kind_qualification_kinds.validities.valid'),
+                                 validity_valid_or_expired: t('activerecord.attributes.event/kind_qualification_kinds.validities.valid_or_expired') } }
+    - entry.grouped_qualification_kind_ids_and_validity('precondition', 'participant').each_with_index do |ids, index|
       .precondition-grouping
         - ids.each do |id|
```

Comment:
Wenn `id` hier mehrere Felder enthält, können diese direkt einzelnen Blockvariablen zugewiesen werden: `ids.each do |id, validity| ... end`. Wenn man `validity` dann nicht verwendet, kann man es auch `_validity` nennen. In jedem Fall wird klar, welche Daten man gerade hat. Insbesondere finde ich es besser zu debuggen als `id[0]`...